android/vcmi-app/src/main/java/eu/vcmi/vcmi/util/LegacyConfigReader.java

```diff
@@ -0,0 +1,100 @@
+package eu.vcmi.vcmi.util;
+
+import android.annotation.TargetApi;
+import android.text.TextUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+
+import eu.vcmi.vcmi.Const;
+
+/**
+ * helper used to retrieve old vcmi config (currently only needed for h3 data path in order to migrate the data to the new location)
+ *
+ * @author F
+ */
+public final class LegacyConfigReader
```

Comment:
First time I see this code. Probably something ancient?

Reply:
probably. it's no longer used.